cannon/testdata/go-1-25/keccak-xorbytes/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"crypto/subtle"
+	"fmt"
+)
+
+func main() {
+	a := make([]byte, 4)
+	p := []byte{1, 2, 3}
+
+	subtle.XORBytes(a, a, p)
```

Comment:
basically, if this doesn't panic then we're good.

Reply:
I don't think this actually panicked in cannon because we don't panic on misaligned memory access. It was here as part of debugging the issue and I forgot to clean up these files.

I've now simplified it down to a single `keccak` test which I've added for all the go versions to confirm that we get the right result from keccak.  Also updated the go 1.25 toolchain to 1.25.4 which was missing before. Conveniently that proved the new keccak test identifies the problems as it fails (wrong hash is calculated) with go 1.25.0 toolchain and passes with go 1.25.4.

Reply:
nice!